lib/line_log/request_data_extractor.rb

```diff
@@ -0,0 +1,35 @@
+module LineLog
+  class RequestDataExtractor
+    def self.call(event, status, began_at)
+      extract_request(event, status, began_at)
+    end
+
+    private
+
+    def self.extract_request(event, status, began_at)
+      data = initial_data(event, status, began_at)
+      data.merge!(custom_options)
+    end
+
+    def self.initial_data(event, status, began_at)
+      {
+        method: event['REQUEST_METHOD'],
+        path: extract_path(event),
+        format: event['HTTP_ACCEPT'],
+        ip: event['HTTP_X_REAL_IP'],
+        status: status,
+        duration: (Time.now - began_at).round(2)
```

Comment:
This must be added to documentation, also specify that it can be overwritten

Together with an example of the resulting log line